truss/templates/server.Dockerfile.jinja

```diff
@@ -45,8 +45,17 @@ RUN pip install -r server_requirements.txt --no-cache-dir && rm -rf /root/.cache
 {{ super() }}
 {% endblock %}
 
+
+{% block app_copy %}
+# Copy data before code for better caching
+{%- if data_dir_exists %}
+COPY ./{{config.data_dir}} /app/data
+{%- endif %}
+
 {% block cache_weights %}
     {%- if config.hf_cache != None %}
+RUN curl -s https://baseten-public.s3.us-west-2.amazonaws.com/bin/b10cp-5fe8dc7da-linux-amd64 -o /app/b10cp; chmod +x /app/b10cp
```

Comment:
we should remove the `{% block cache_weights %}`. idk how nested blocks work in jinja and would rather avoid it